CefSharp.Core/Internals/RenderClientAdapter.h

```diff
@@ -167,7 +167,11 @@ namespace CefSharp
                     bitmapInfo->NumberOfBytes = numberOfBytes;
                 }               
 
-                CopyMemory(backBufferHandle, (void*)buffer, bitmapInfo->NumberOfBytes);
+                if (!bitmapInfo->DirtyRectSupport || ((bitmapInfo->DirtyRect.Width != 0) && (bitmapInfo->DirtyRect.Height != 0)))
```

Comment:
I tried to reproduce and couldn't, the code remains correct without it,
please ignore/remove that change.

On Wed, Nov 4, 2015 at 10:43 AM, Alex Maitland <notifications@github.com>
wrote:

> In CefSharp.Core/Internals/RenderClientAdapter.h
> <https://github.com/cefsharp/CefSharp/pull/1349#discussion_r43859227>:
>
> > @@ -167,7 +167,11 @@ namespace CefSharp
> >                      bitmapInfo->NumberOfBytes = numberOfBytes;
> >                  }
> >
> > -                CopyMemory(backBufferHandle, (void*)buffer, bitmapInfo->NumberOfBytes);
> > +                if (!bitmapInfo->DirtyRectSupport || ((bitmapInfo->DirtyRect.Width != 0) && (bitmapInfo->DirtyRect.Height != 0)))
>
> @bartwe <https://github.com/bartwe> Under what scenario are you seeing Width
> == 0 and Height == 0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cefsharp/CefSharp/pull/1349/files#r43859227>.
>